os/arch/arm/src/armv7-a/arm_initialize.c

```diff
@@ -101,7 +102,7 @@ void up_initialize(void)
 {
 #ifdef CONFIG_SYSTEM_REBOOT_REASON
 	up_reboot_reason_init();
-	lldbg("[Reboot Reason] : %d\n", up_reboot_reason_read());
+	lldbg("[Reboot Reason] : %d [BKUP_REG2] 0x%x \n", up_reboot_reason_read(), BKUP_Read(BKUP_REG2));
```

Comment:
This is a common code, not Realtek specific. So, we can't add it here.
It seems you print it in the rtl8730e_boot.c. Do you need to do here again?

Reply:
Oh, it's the timer to check km4 status. Anyway, we can't add this log here. Please move it into the realtek function.

Reply:
Move to bootloader side to display it.

![image](https://github.com/user-attachments/assets/2cb01cd3-2233-4ea8-934e-e0a09c037ee0)